### PR TITLE
achievement unlocked by JAremko

### PR DESCRIPTION
**Pull request by d12frosted**
Seriously.

### CODE REVIEW

**Comment by JAremko** [comment]
Are we talking about something like the "Golden Raspberry"  award?  :smile: 

I hope it's not a :banana:

File: doc/DOCUMENTATION.org [hunk 1]
```diff
@@ -2818,6 +2818,7 @@ with ~SPC SPC spacemacs/recompile-elpa~.
 | The Expert in Latin Language ([[https://github.com/syl20bnr/spacemacs/pull/4043][PR]])               | [[https://github.com/vijaykiran][vijaykiran]]   |
 | The Tiler ([[https://github.com/syl20bnr/spacemacs/pull/4068][eyebrowse integration]])               | [[https://github.com/bmag][bmag]]         |
 | The Geometer ([[https://github.com/TheBB/spaceline][spaceline]])                        | [[https://github.com/TheBB][TheBB]]        |
+| The Emotional                        | [[https://github.com/JAremko][JAremko]]        |
```
Comment (JAremko) [inline doc/DOCUMENTATION.org]: But I simply drink too much coffee :confused:

**Comment by syl20bnr** [comment]
I agree for an achievement but maybe something different :-)
Also I hijack this post for @StreakyCobra who desserve a special achievement too.

File: doc/DOCUMENTATION.org (hunk 1 above)
Comment (robbyoconnor) [inline doc/DOCUMENTATION.org]: This community would not be the same without @JAremko

**Comment by joehillen** [comment]
I imagine @StreakyCobra would be something like "The Curator"

**Comment by d12frosted** [comment]
@syl20bnr I agree that @StreakyCobra deserves a special achievement. 
It's just, @JAremko makes me laugh every time he posts ascii emotions :)

**Comment by JAremko** [comment]
>It's just, @JAremko makes me laugh every time he posts ascii emotions :)

If I am to be remembered as the one who brought the Twitch chat plague upon the Spacemacs community...  I shall be known as  "웃웃__̴ı̴̴̡̡̡ ̡͌l̡̡̡ ̡͌l̡*̡̡ ̴̡ı̴̴̡ ̡̡͡|̲̲̲͡͡͡ ̲▫̲͡ ̲̲̲͡͡π̲̲͡͡ ̲̲͡▫̲̲͡͡ ̲|̡̡̡ ̡ ̴̡ı̴̡̡ ̡͌l̡̡̡̡웃 The Twitch chat fugitive ==웃"   

>Also I hijack this post for @StreakyCobra who desserve a special achievement too.

Lets brain storm diz 

I see I seee `the tagger` `the tag vending machine`... `the streaky tagger`... 
`The One Who Tags (•_•) / ( •_•)>⌐■-■ / (⌐■_■),`

**Comment by joehillen** [comment]
* The Tagginator ("Come with me if you want your issues tagged.")
* Sir Tags-A-Lot

**Comment by d12frosted** [comment]
The Issuepedia?

**Comment by d12frosted** [comment]
I mean, he also knows a lot about existing issues, not only tags them :)

**Comment by d12frosted** [comment]
@StreakyCobra common, take part in this conversation!

**Comment by StreakyCobra** [comment]
None of my business, I'll unsubscribe from this thread to let you discuss freely :wink:

**Comment by sdwolfz** [comment]
I resurrect this PR and propose: `The ASCII Scribe`!

(will take over and open a new PR but merge it as a contribution from @d12frosted)

**Comment by d12frosted** [comment]
@sdwolfz did the same with this PR

**Comment by sdwolfz** [comment]
Thank you :heart:!
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**